mitodl/ocw-studio
# Link to Collections

#### Pre-Flight checklist


#### What are the relevant tickets? #1038 

#### What's this PR do?
Adds support for linking to course collections and resource collections via the CKEditor "Add Link" tool.

#### How should this be manually tested?
Try adding resource and course collections to a page. Click "Save" and verify that the markdown is saved as expected, i.e., that shortcodes of the form `{{< resource_link some-collection-uuid "Some Title" >}} are added to the markdown with the correct UUID. 

#### Screenshots
<img width="711" alt="Screen Shot 2022-02-22 at 2 31 35 PM" src="https://user-images.githubusercontent.com/9010790/155206117-d5681d5b-4dec-488b-aad4-a220c68bff10.png">
